backend/wmg/data/transform.py

```diff
@@ -206,7 +206,7 @@ def _cell_type_ordering_per_tissue(data_cells: pd.DataFrame, target_cells: List[
 
 
 @log_func_runtime
-def generate_primary_filter_dimensions(snapshot_path: str, corpus_name: str, snapshot_id: int):
```

Comment:
`corpus_name` was never set to anything different than the `snapshot_path` so I removed it. This simplifies the access pattern in the processing pipeline, as well.